bin/recreate_database.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+python -m bin.lib.recreate_database
```

Comment:
Just a suggestion, it might be useful to include a `syncdb` command and possibly migrate the schema as well, since the user is most likely going to want to do those things immediately after running this script anyway.